### PR TITLE
jsx-wrap-multilines check arrow functions without block body

### LINKED ISSUE #790: jsx-wrap-multilines to check stateless components
Given this code, check should not pass

```
const Grade = ({ grade }) =>
  <View style={styles.gradeContainer}>
    <Text style={styles.grade}>{grade}</Text>
  </View>;
```

For this code, check should pass:

```
const Grade = ({ grade }) => (
  <View style={styles.gradeContainer}>
    <Text style={styles.grade}>{grade}</Text>
  </View>
);
```

### PR DESCRIPTION
**Pull request by ColCh**
Aka "lambdas"

Close #790

I will duplicate issue desc here:

Given this code, check should not pass

```jsx
const Grade = ({ grade }) =>
  <View style={styles.gradeContainer}>
    <Text style={styles.grade}>{grade}</Text>
  </View>;
```

For this code, check should pass:

```jsx
const Grade = ({ grade }) => (
  <View style={styles.gradeContainer}>
    <Text style={styles.grade}>{grade}</Text>
  </View>
);
```

I leaved this behaviour by-default, but it can break backward compatibility?

May be It should be turned off by default?

### CODE REVIEW

**Comment by yannickcr** [comment]
Thanks for this.

> I leaved this behaviour by-default, but it can break backward compatibility?
> May be It should be turned off by default?

If I'm not mistaken, according to [ESLint Semantic Versioning Policy](https://github.com/eslint/eslint#semantic-versioning-policy) this is a breaking change, so I'll wait for the next major release to merge it.

**Comment by ColCh** [comment]
okay, thanks!

File: tests/lib/rules/jsx-wrap-multilines.js [hunk 1]
```diff
@@ -73,6 +73,18 @@ var ASSIGNMENT_NO_PAREN = '\
...
+
+var ARROW_PAREN = '\
+  var hello = () => (<div>\n\
+    <p>Hello</p>\n\
+  </div>);';
+
+var ARROW_NO_PAREN = '\
+  var hello = () => <div>\n\
+    <p>Hello</p>\n\
+  </div>;';
```
Comment (lencioni) [inline tests/lib/rules/jsx-wrap-multilines.js]: @ljharb After your Node 4+ PR this could get cleaned up a little.

Comment (ljharb) [inline tests/lib/rules/jsx-wrap-multilines.js]: Good call; I'll include this in my PR.

**Pull request merged**